docs/release-notes.txt

```diff
@@ -4,6 +4,115 @@
+ Version 0.17
+**************
+
+Version 0.17.0
+==============
+
+**Release Date:** September 28, 2021
+
+**New Features**
+
+-  Support the `MMDetection <https://github.com/open-mmlab/mmdetection>`__ library in Model Hub to
+   easily train object detection models. MMDetection provides efficient implementations of popular
+   object detection methods like Mask R-CNN, Faster R-CNN, and DETR on par with Detectron2. In
+   addition, cutting-edge approaches from academia are regularly added to the library.
+
+-  Add a new ``--auto-work-dir`` feature to ``det deploy local``. Setting ``--auto-work-dir
+   /some/path`` will have two effects: first, ``/some/path`` will be bind-mounted into the container
+   (still as ``/some/path``); second, interactive jobs (notebooks, shells, and commands) will run in
+   the provided working directory by default. Note that containers run as the root user by default,
+   so you may want to configure your user with ``det user`` such that interactive jobs run as your
+   regular user.
+
+-  Add the ability to use customizable master configuration templates in ``det deploy aws|gcp``.
+
+-  WebUI: Allow experiment owners to delete their own experiments.
+
+   .. image:: https://user-images.githubusercontent.com/220971/134048799-cd663a75-cb24-4f44-9a8a-c2ff23222cef.png
+
+-  Support configuring an idle timeout for TensorBoards and notebooks.
+
+-  Images: Add an environment image for CPU-only TensorFlow 2.5 and 2.6.
+
+-  WebUI: Add support for batch delete on the experiment list page.
+
+   .. image:: https://user-images.githubusercontent.com/220971/133659677-aea0d1bc-95ce-4652-8218-92b97d114358.png
+
+-  Support configuring the working directory using the ``work_dir`` configuration field for
+   commands, shells, and notebooks. You can also optionally set it in the
+   ``task_container_defaults.work_dir`` field of the master configuration. The value set in the
+   master configuration will be ignored when a context directory is submitted.
+
+-  WebUI: Display the latest log entry available for a trial at the bottom of the trial's page. This
+   works for both single-trial experiments and trials within a multi-trial experiment.
+
+   .. image:: https://user-images.githubusercontent.com/220971/131391658-4be1a1f4-1d46-4766-a737-7eb8efcb65b4.png
+
+-  WebUI: Add support for displaying NaN and Infinity metric values.
+
+**Improvements**
+
+-  API: The aggregated historical resource allocation APIs
+   ``/api/v1/resources/allocation/aggregated`` and ``/allocation/aggregated`` now account for all
+   resources, not just those allocated to experiments.
+
+-  Images: Add CPU-only images for TF 2.5 and 2.6.
+
+-  Images: TF 2.5 and 2.6 images will no longer include PyTorch builds. For PyTorch 1.9, please use
+   the combined TF 2.4/PyTorch 1.9 image.
+
+-  Images: TF 2.4, 2.5, 2.6, and PyTorch 1.9 images will now use Python 3.8. The legacy TF
+   1.15/PyTorch 1.7 image will continue to use Python 3.7.
+
+**Changes**
```

Comment:
non-blocking: May make sense to put all breaking changes at the very top, even if they are a mix of "changes" and "bug fixes" etc.

Reply:
Hmm, sort of makes sense, but we haven't done that before. I'll ponder it.

Reply:
Yeah, sure, why not. New minor version, new world.